src/components/login/Registration.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { TextField , Button , Box , Alert , FormControlLabel  , Checkbox } from "@mui/material";
+import { useState } from "react"
+import {  useNavigate } from 'react-router-dom';
+
+export const Registration = () => {
+     const [error , setError] = useState({
+        status : false,
+        msg:"",
+        type:""
+    })
+
+    const navigate = useNavigate();
+
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        const data = new FormData(e.currentTarget);
+        const actualData = {
+            email: data.get("email"),
+            password: data.get("password"),
+        }
+        
+        if(actualData.email && actualData.password){
+            console.log(actualData)
+              document.getElementById('login-form').reset()
```

Comment:
Why are you directly manipulating the DOM?

Reply:
to blank the input